src/System.IO.Pipelines.Text.Primitives/ReadableBufferExtensions.cs

```diff
@@ -149,6 +151,31 @@ public unsafe static ulong GetUInt64(this ReadableBuffer buffer)
             return value;
         }
 
+        public unsafe static string GetAsciiString(this Span<byte> span)
```

Comment:
@KrzysztofCwalina Are you referring to the span => Ascii string conversion? We'd need to add an Ascii encoder, but it should be pretty fast / easy.